build.gradle

```diff
@@ -115,6 +117,8 @@ dependencies {
     libraries.mockito,
     libraries.truth,
     libraries.toolsFxTesting
+
+  apt libraries.autovalue
```

Comment:
I made the changes in build.gradle to add support for AutoValue generated files in Intellij, but you are right, that particular line was not necessary. Removing it.